lib/recase/cases/snake_case.ex

```diff
@@ -4,27 +4,22 @@ defmodule Recase.SnakeCase do
 
   This module should not be used directly.
 
+  ## Examples
+
+      iex> Recase.to_snake "foo_barBaz-λambdaΛambda-привет-Мир"
```

Comment:
Can we also provide ascii examples as well? It would we strange to see for people who do not need unicode support.

I guess, that the right formula is: one ascii-only example and one unicode-friendly example.

Reply:
We surely can, but correct me if I am wrong, these `Recase.Kind.convert/1` were never supposed to be called directly and the respective `Recase.to_kind/1` docs have _ascii_ example.

So I am confused. Do you want me to copy the example from there? Provide some newly invented example?

Reply:
Yes, that's fine. It was just a misread from me.

Reply:
I have literally confused two APIs.

Reply:
TBH, you invented 2 APIs :)